BotrAPI.php

```diff
@@ -1,28 +1,109 @@
 <?php
 
-/*-----------------------------------------------------------------------------
- * PHP client library for Bits on the Run System API
+/**
+ * This file contains an API client for the Bits on the Run API version 1.4
+ *
+ * PHP version 5, 7
+ *
+ * For the System API documentation see:
+ * http://developer.longtailvideo.com/botr/system-api/
+ *
+ * LICENSE:
  *
- * Author:      Sergey Lashin
- * Copyright:   (c) 2012 LongTail Ad Solutions
- * License:     BSD 3-Clause License
- *              See accompanying LICENSE file
+ * Copyright 2012 LongTail Ad Solutions
  *
- * Version:     1.4
- * Updated:     Wed Feb  8 11:59:56 CET 2012
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @category  Services
+ * @package   BotrAPI
+ * @author    Sergey Lashin <support@jwplayer.com>
+ * @copyright 2012 LongTail Ad Solutions
+ * @license   BSD 3-Clause License
+ * @link      https://github.com/silverorange/botr-api
+ */
+
+/**
+ * PHP client library for Bits on the Run System API
  *
  * For the System API documentation see:
- *              http://developer.longtailvideo.com/botr/system-api/
- *-----------------------------------------------------------------------------
+ * http://developer.longtailvideo.com/botr/system-api/
+ *
+ * @category  Services
```

Comment:
Yeah. PEAR standard requires file-level and class-level documentation with those tags.